apps/admin/frontend/src/screens/tally_report_screen.tsx

```diff
@@ -363,12 +350,10 @@ export function FullElectionTallyReportScreen(): JSX.Element {
 export function AllPrecinctsTallyReportScreen(): JSX.Element {
   const { electionDefinition, isOfficialResults } = useContext(AppContext);
   assert(electionDefinition);
-  const { election } = electionDefinition;
 
   const reportResultsQuery = getResultsForTallyReports.useQuery({
     groupBy: {
       groupByPrecinct: true,
-      groupByParty: electionHasPrimaryContest(election),
```

Comment:
Previously it was up to the frontend to tell the backend whether it wanted things grouped by party - now we default to doing that on the backend if it's a primary. Moving it as a default to the backend avoids the footgun of forgetting to do this for new pages that use this query.